scripts/ebtc_governance.py

```diff
@@ -0,0 +1,220 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from rich.console import Console
+
+C = Console()
+
+"""
+The following scripts are meant to provide a convenient access to the multisig governance operations of the EBTC ecosystem.
+These script will use the default multisig intended for each of the timelocks. In other words, operations interacting with
+the Low Sec Timelock will TechOps multisig and those interacting with the High Sec Timelock will use the Ecosystem multisig.
+
+NOTE: Operations interacting with timelock will need to be executed twice. Once to queue the transaction and once to execute
+it at the end of the delay period.
+"""
+
+
+"""
+Low Sec Governance Operations
+
+The following operations are meant to be executed by the TechOps multisig.
+"""
+
+
+def cdpManager_set_staking_reward_split(value):
+    """
+    Set the staking reward split for the CDPManager contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.techops_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for CDPManager.setStakingRewardSplit\n")
+    safe.ebtc.cdpManager_set_staking_reward_split(value)
+
+    safe.post_safe_tx()
+
+
+def cdpManager_set_redemption_fee_floor(value):
```

Comment:
will have the same suggestion here as for `cdpManager_set_staking_reward_split`. ideally should be the same protective programming for all the methods which are here in `ebtc_governance` and onchain they have an interval that should be expected

Reply:
Addressed https://github.com/ebtc-protocol/ebtc-multisig/commit/157869200187017c7f7fd3be2308e35f1c8c04ac